### PR TITLE
Branches - Julia Bouvier

### PR DESCRIPTION
**Pull request by juliabouv**
# Binary and Decimal
Congratulations! You're submitting your assignment.

### CODE REVIEW

File: test/binary_to_decimal_test.rb [hunk 1]
```diff
@@ -2,32 +2,57 @@
...
     binary_array = Array.new(8) { rand(0..1) }
     expected_decimal = binary_array.join.to_s.to_i(2)
-
+    
     binary_to_decimal(binary_array).must_equal expected_decimal
   end
 end
+
+describe "decimal to binary" do
+  it "From 153 to 10011001" do
```
Comment (CheezItMan) [inline test/binary_to_decimal_test.rb:38]: I would also maybe do a test for what happens with `[0]`, or an empty array.

**Review by CheezItMan — COMMENTED** [review]
Well done!  You hit the learning goals plus the extras!
